.github/workflows/ci.yml

```diff
@@ -28,7 +28,7 @@ jobs:
         with:
           python-version: "3.10"
       - run: ./setup.sh
-      - run: rustup component add rustfmt --toolchain nightly
-      - run: rustup component add clippy --toolchain nightly
+      - run: rustup component add rustfmt
+      - run: rustup component add clippy
```

Comment:
We shouldn't need this if they are in the rust-toolchain file.

Reply:
Ah right, I only needed it before since I wanted to run these specifically as nightly. Let's try without them.